_assets/css/_settings.scss

```diff
@@ -17,13 +17,15 @@
 //  12. Table
 
 @import 'util/util';
-// @import 'https://code.cdn.mozilla.net/fonts/fira.css';
+
+// Inter Font
+@import url('https://rsms.me/inter/inter.css');
```

Comment:
@muffinresearch The reason we're making the switch to Inter here is because there's a big re-brand coming for the org. We've talked with John Gruen and Sean Martell who are both on the re-branding team and they mentioned that Fira will be retired with the new re-brand, so we're getting in front of it now with Extension Workshop. 

One of the complications is that Inter requires a license which AFAIK is held by the Mozilla brand team. Unfortunately I am not technically savvy enough to know if there's another way to get the font without referencing from the 3rd party.

Reply:
@caitmuenster Thanks for the info. I think the details are here: https://rsms.me/inter/ including the license which is here: https://choosealicense.com/licenses/ofl-1.1/

Looks like the download from there contains the web fonts + CSS so we can look to host it  ourselves - I don't think self-hosting would change much license-wise, I would think the same restrictions would govern both, but I'm not a lawyer 😅

Reply:
I don't think it will be a problem, either, but I'll confirm with the brand folks. :)

Reply:
Ok, I've moved to self-hosting the Inter font.